garage/replay_buffer/simple_replay_buffer.py

```diff
@@ -25,16 +25,16 @@ def sample(self, batch_size):
             buffer[key] = self._buffer[key][:self._current_size]
```

Comment:
you can remove the constructor on this class -- it is equivalent to inheritance with no overriding constructor.